utils/model.py

```diff
@@ -307,6 +307,19 @@ def completion_stream(self, messages: list[dict[str, str]], generation_config: G
                 is_print_speed):
             yield output, finish_reason
 
+    def completion_stream(self, prompt: str, generation_config: GenerationConfig,
```

Comment:
I am not quite in favor of `overloading` (especially in dynamic lang like python), maybe we can use another name?

Reply:
Yes. You are right